### ISSUE TITLE
Research audio players

### ISSUE THREAD

**Issue by amcquade** [amcquade/quick-feed#3]
User should be able to play episodes from the feed listed on the feed. Suggestions can go below

**Comment by danielgrijalva** [comment]
It may be simple, but the [`<audio />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio) tag can be used for this.

**Comment by amcquade** [comment]
@danielgrijalva was looking at [react-player](https://www.npmjs.com/package/react-player) the other day, havent had the time to play around with it.

**Comment by danielgrijalva** [comment]
@amcquade that looks way better! I can try to implement it if you want. Do you have any mock ups/design ideas on how to display the player?

**Comment by amcquade** [comment]
@danielgrijalva not really, although lately ive been thinking that the soundcloud player is something wed like to try to emulate. a player containing a list of episodes instead of creating players/elements for each episode

**Comment by danielgrijalva** [comment]
I see... like this?
![image](https://user-images.githubusercontent.com/11547406/56428770-6fecdb00-6275-11e9-9c7f-a04cc9aebebc.png)
That's much more user friendly :ok_hand: I'll see what I can do.

**Comment by danielgrijalva** [comment]
Hey @amcquade, sorry for the delay. Here's my progress so far:
![Peek 2019-05-06 11-21](https://user-images.githubusercontent.com/11547406/57246076-3ea03900-6ff1-11e9-9c96-416812575340.gif)

What do you think? The actual player is from `react-player` and I'm looking to change its CSS to make it look like this:

![image](https://user-images.githubusercontent.com/11547406/57246209-8f179680-6ff1-11e9-8094-f12e8a94be04.png)

**Comment by amcquade** [comment]
@danielgrijalva no worries, looks great ! my only suggestion being some kind of wrapping the list in some sort of container that enables scrolling for the list and not the whole page
